### PR TITLE
Provides conan recipe file to artifacts

### PR DESCRIPTION
**Pull request by deekshith-elear**
Adds conan recipe file

### CODE REVIEW

File: conanfile.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
```
Comment (manavkumarm) [inline conanfile.py:1]: In general, in all conan files, let's keep exactly one blank line between sections, and one single space between multiple arguments separated by commas (,)

File: conanfile.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        self.run("cd .. && autoreconf -fsi ")
+        autotools.configure(configure_dir="..",args=["--prefix=${PWD}"])
+        autotools.make()
+        autotools.install()
+    def package(self):
+        self.copy("*.h", dst="include", src="include/include")
+        self.copy("*.la", dst="lib", src="lib",keep_path=False)
+
+    def package_info(self):
+        self.cpp_info.libs = ["catta"]
```
Comment (manavkumarm) [inline conanfile.py]: In all conan files, keep a space after "[" and a space before "]" in such declarations

**Review by manavkumarm — CHANGES_REQUESTED** [review]

File: conanfile.py (hunk 2 above)
Comment (deekshith-elear) [reply conanfile.py]: handled

File: conanfile.py (hunk 1 above)
Comment (deekshith-elear) [reply conanfile.py:1]: handled

**Pull request closed without merging**